backend/tests/services/test_external_events.py

```
@@ -0,0 +1,1956 @@
+            for round_index in range(rounds):
+                buffer.append(
+                    "gitlab",
+                    "acme/app!7",
+                    "merged",
+                    {
+                        "event_id": f"{index}-{round_index}",
+                        "summary": "merged",
+                        "event_type": "merged",
+                    },
+                )
+        except BaseException as exc:  # pragma: no cover - failure surface
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `BaseException`

(BLE001)